indico/modules/designer/models/templates.py

```diff
@@ -115,6 +118,11 @@ def __init__(self, **kwargs):
     def owner(self):
         return self.event_new if self.event_new else self.category
 
+    @property
+    def background_image(self):
+        image_id = self.data['background']['image_id']
+        return DesignerImageFile.get(int(image_id)) if image_id else None
```

Comment:
Why are we storing the image id as a string? Actually, why are we storing the `image_id` in the JSON at all and not in a column as before?